src/Concordium/Client/Output.hs

```diff
@@ -719,11 +719,6 @@ showRejectReason verbose = \case
   Types.RejectedReceive{..} ->
     let (contractName, funcName) = Wasm.contractAndFunctionName receiveName
     in [i|'#{funcName}' in '#{contractName}' at #{showCompactPrettyJSON contractAddress} failed with code #{rejectReason}|]
-  Types.NonExistentRewardAccount a ->
```

Comment:
Just to be sure - this is obsolete (and always has been on mainnet) ?

Reply:
Correct. This was used a long time ago when a baker, when registering, would designate an account to be rewarded on.

This was removed at least for testnet 4 (which was the testnet around 6 months before mainnet).